scripts/utils/setup-roles.ts

```diff
@@ -47,24 +50,24 @@ export async function main(
     whitelist: Contract,
 ): Promise<void> {
     const signers: SignerWithAddress[] = await hre.ethers.getSigners();
-    const [deployer, admin, adminMultiSig] = signers;
+    const [deployer] = signers;
 
     console.log("Deployer address:", deployer.address);
     // Get deployer balance
     const provider = ethers.provider;
     const balance = await provider.getBalance(deployer.address);
     console.log("Deployer balance:", balance.toString());
 
-    console.log("Admin address:", admin.address);
-    console.log("Admin MultiSig address:", adminMultiSig.address);
+    // Set admin address
+    const ADMIN_ADDRESS = process.env.ADMIN_MULTISIG;
+    console.log("Admin address:", ADMIN_ADDRESS);
 
     // Define roles
     const ADMIN_ROLE = DEFAULT_ADMIN_ROLE;
```

Comment:
We rename `ADMIN_ROLE` to `DEFAULT_ADMIN_ROLE` above (line 13), then rename it again in this assignment. Same for the other roles here. Seems unnecessary?

Reply:
Importing them from `constants.ts` and defining them in the import statement is not sufficient and it throws an error.  They need to be defined in the scope of the function with their aliases.

Reply:
What's the error you get? Just thinking about how JavaScript works, I'm not sure why this would make sense. If they're defined in the global scope they are also defined in the function scope.

Reply:
When I comment them out on lines 66 - 69, this is generated:

![Screen Shot 2022-06-15 at 5 22 56 PM](https://user-images.githubusercontent.com/42501101/173932271-77e0a167-c397-43cc-9786-04497b9f3641.png)

Reply:
Well, that's cause you are still aliasing them with `as` in the imports. I'd change lines 8-12 to:

```
import {
    ORIGINATOR_ROLE,
    ADMIN_ROLE,
    FEE_CLAIMER_ROLE,
    REPAYER_ROLE,
} from "./constants";
```

the whole renaming thing just seems unnecessary, adding `DEFAULT_` doesn't really give the reader of the script any useful information.

Reply:
I am sorry if I am missing something here...

Upon removing the `as` in the imports as shown in your comment, the role definitions start erroring per the screenshot.

![Screen Shot 2022-06-16 at 11 31 39 AM](https://user-images.githubusercontent.com/42501101/174106338-3c5effc3-d9cc-4fd6-ae78-d1eac22da15f.png)

Reply:
Yep, you need to also remove lines 66-69....I just tried it locally and all works. Lines 11-16 should be:

```
import {
    ORIGINATOR_ROLE,
    ADMIN_ROLE,
    FEE_CLAIMER_ROLE,
    REPAYER_ROLE,
} from "./constants";
```